include/quicly/loss.h

```diff
@@ -248,16 +197,9 @@ inline int quicly_loss_on_alarm(quicly_loss_t *r, uint64_t largest_sent, uint64_
         *num_packets_to_send = 0;
         return quicly_loss_detect_loss(r, largest_acked, do_detect);
     }
-    if (r->tlp_count < r->conf->max_tlps) {
-        /* Tail Loss Probe. */
-        r->tlp_count++;
-        *num_packets_to_send = 1;
-        return 0;
-    }
-    /* RTO */
-    if (r->rto_count == 0)
-        r->largest_sent_before_rto = largest_sent;
-    ++r->rto_count;
+    /* PTO */
+    if (r->pto_count == 0)
+        ++r->pto_count;
     *num_packets_to_send = 2;
```

Comment:
Do you think it's reasonable to keep the API that returns the maximum number of packets that should be sent? I ask this because I think there's a potential of changing this to a boolean value indicating if any packet should be sent.

Reply:
Yeah, I was thinking about that, but it fell off my radar. I can do that.